commons/src/main/java/org/apache/mesos/elasticsearch/common/zookeeper/ZookeeperCLIParameter.java

```diff
@@ -20,4 +20,12 @@ public String getZookeeperUrl() {
     public long getZookeeperTimeout() {
         return zookeeperTimeout;
     }
+
+    public static final String ZOOKEEPER_FRAMEWORK_URL = "--zookeeperFrameworkUrl";
```

Comment:
So will that mean that both zookeeperMesosURL and zookeeperFrameworkURL are required? Or just one? If so, which one?
It doesn't make sense that the user should have to set both to the same value.